tools/packaging/kata-deploy/runtimeclasses/kata-rustRuntimeClasses.yaml

```diff
@@ -0,0 +1,26 @@
+---
+kind: RuntimeClass
+apiVersion: node.k8s.io/v1
+metadata:
+    name: kata-cloud-hypervisor
```

Comment:
Random thought: I know the class is specified in the filename, but since YAML supports comments, how about we add one stating explicitly that `cloud-hypervisor` only refers to CH with the rust runtime.

Alternatively, we could add a `metadata.description` that could do the same job but be programmatically queryable.

Reply:
I like this idea a lot!  I will add that, @jodh-intel!

Reply:
I had to drop the metadata.description bits as those are not supported by k8s, causing the runtime-class to fail to be applied.